doc/source/management/data_at_rest_encryption.rst

```diff
@@ -363,33 +683,73 @@ System Variables
 
 .. variable:: keyring_vault_config
 
-  :version 5.7.20-18: Implemented
-  :cli: ``--keyring-vault-config``
-  :dyn: Yes
-  :scope: Global
-  :vartype: Text
-  :default:
+   :version 5.7.20-18: Implemented
+   :cli: ``--keyring-vault-config``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Text
+   :default:
 
 This variable is used to define the location of the
 :ref:`keyring_vault_plugin` configuration file.
 
 .. variable:: keyring_vault_timeout
 
-  :version 5.7.21-20: Implemented
-  :cli: ``--keyring-vault-timeout``
-  :dyn: Yes
-  :scope: Global
-  :vartype: Numeric
-  :default: ``15``
+   :version 5.7.21-20: Implemented
+   :cli: ``--keyring-vault-timeout``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Numeric
+   :default: ``15``
 
 This variable allows to set the duration in seconds for the Vault server
 connection timeout. Default value is ``15``. Allowed range is from ``1``
 second to ``86400`` seconds (24 hours). The timeout can be also completely
 disabled to wait infinite amount of time by setting this variable to ``0``.
 
-Other reading
--------------
+Data Scrubbing
+================================================================================
+
+While data encryption ensures that the existing data are not reachable in
+plain form, the data scrubbing removes the data once the user decides they
+should be deleted. Also, data scrubbing removes copies of data from logs.
+
+Once enabled, data scrubbing works automatically on each tablespace
+separately. To enable data scrubbing, you need to set the following variables:
+
+- :variable:`innodb-background-scrub-data-uncompressed`
+- :variable:`innodb-background-scrub-data-compressed`
+
+Uncompressed tables can also be scrubbed immediately, independently of key
+rotation or background threads. This can be enabled by setting the variable
+innodb-immediate-scrub-data-uncompressed. This option is not supported for
+compressed tables.
+
+System Variables
+--------------------------------------------------------------------------------
+
+.. variable:: innodb_background_scrub_data_compressed
+
+   :version 5.7.23-24: Implemented
+   :cli: ``--innodb-background-scrub-data-compressed``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Boolean
+   :default: ``OFF``
+
+.. variable:: innodb_background_scrub_data_uncompressed
+
+   :version 5.7.23-24: Implemented
+   :cli: ``--innodb-background-scrub-data-uncompressed``
+   :dyn: Yes
+   :scope: Global
+   :vartype: Boolean
+   :default: ``OFF``
+
+
+.. seealso::
 
-* `Vault Documentation <https://www.vaultproject.io/docs/index.html>`_
-* `General-Purpose Keyring Key-Management Functions
-  <https://dev.mysql.com/doc/refman/5.7/en/keyring-udfs-general-purpose.html>`_
+   Vault Documentation
```

Comment:
Some of the features link to MariaDB docs, some don't, here e.g. https://mariadb.com/kb/en/library/xtradb-innodb-data-scrubbing/

Reply:
@laurynas-biveinis It is not clear how the documentation must be updated in respect to your comment.

Reply:
How do you think it should be? Should we link to all applicable MariaDB docs or shouldn't we?

Reply:
In text, there is no reference how we use MariaDB implementation. The `see also` block implies that it is "useful reading related to the subject".  I would add more information about MariaDB if we decide to explain the implementation more. in any case it appears to be a separate problem not quite related to the current context of release notes.